Do not wrap yast2 arguments with double quotes

ge0r opened the pull request:
When passing a string that contains words separated by spaces as argument to the `yast2_console_exec` function, wrapping it with double quotes leads to yast2 error, unknown command. 
Not wrapping the argument with double quotes mitigates the issue.

- Related ticket: No ticket
- Needles: No needles
- Verification run: 
example when passing argument `modules target=compact` : [previous run](https://openqa.suse.de/tests/8391905#step/clone/5) (failing) | [verification run](https://openqa.suse.de/tests/8392443#step/clone/5) | [verification run TW](https://openqa.opensuse.org/tests/2285520#details)

jknphy reviewed (commented):
seems that I introduced this line to escape quotes in #11156
please, run verification for [yast_oneclick_install](https://openqa.opensuse.org/tests/2284048#step/oneclick_install/10)
to be sure that we don't break anything when passing [$url_ymp](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/cae6400303ef59c58de143fbaf3050bf5e22457e/tests/console/oneclick_install.pm#L20)

ge0r commented:
> seems that I introduced this line to escape quotes in #11156 please, run verification for [yast_oneclick_install](https://openqa.opensuse.org/tests/2284048#step/oneclick_install/10) to be sure that we don't break anything when passing [$url_ymp](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/cae6400303ef59c58de143fbaf3050bf5e22457e/tests/console/oneclick_install.pm#L20)

Good point, I updated the description with the verification run of https://openqa.opensuse.org/tests/2285520#details . Removing the quotes works fine for passing the url as well.
